app/controllers/claims_controller.rb

```diff
@@ -133,7 +133,10 @@ def start_new
       clear_claim_session
       redirect_to(new_claim_path(current_journey_routing_name))
     else
-      redirect_to(claim_path(current_claim.policy.routing_name, slug: slug_to_redirect_to))
+      new_journey = Journeys.for_policy(current_claim.policy)
+      new_claim_slug_sequence = new_journey.slug_sequence.new(current_claim)
+      new_page_sequence = PageSequence.new(current_claim, new_claim_slug_sequence, session[:slugs], params[:slug])
+      redirect_to(claim_path(new_journey::ROUTING_NAME, slug: new_page_sequence.next_required_slug))
```

Comment:
A lot is happening here; I wonder if these statements could be extracted to a separate method, e.g. `redirect_to_new_journey_page`. Just an idea

Reply:
Thanks, done.